gui/browserv7/src/RBrowser.cxx

```diff
@@ -452,7 +452,9 @@ std::string ROOT::Experimental::RBrowser::ProcessDblClick(const std::string &ite
 
       canv->ForceUpdate(); // force update async - do not wait for confirmation
 
-      return "";
+      std::string res = "SLCTCANV:";
+      res.append(canv->GetName());
+      return res;
```

Comment:
While we using string_literals, one can simply do:
```
return "SLCTCANV:"s + canv->GetName();
```

Reply:
Solved in : fe09b4c